src/map/pet.hpp

```diff
@@ -165,6 +201,7 @@ struct pet_data {
 	struct pet_skill_attack* a_skill;
 	struct pet_skill_support* s_skill;
 	struct pet_loot* loot;
+	std::shared_ptr<s_petautobonus> autobonus, autobonus2, autobonus3;
```

Comment:
Is there a reason why we only allow one autobonus for pets and a whole vector for players?